arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift

```diff
@@ -31,26 +31,23 @@ class DistanceMeasurementAnalysisViewController: UIViewController, AGSGeoViewTou
     
     required init?(coder: NSCoder) {
         // Create the scene.
-        scene = AGSScene(basemap: .imagery())
+        scene = AGSScene(basemap: .topographic())
         
-        /// The url of the Terrain 3D ArcGIS REST Service.
-        let worldElevationServiceURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
-        /// The url of the image service for elevation in Brest, France.
-        let brestElevationServiceURL = URL(string: "https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer")!
+        // The elevation image service URL.
+        let elevationServiceURL = URL(string: "https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer")!
         
         // Create the surface and set it as the base surface of the scene.
-        let elevationSources = [
-            AGSArcGISTiledElevationSource(url: worldElevationServiceURL),
-            AGSArcGISTiledElevationSource(url: brestElevationServiceURL)
-        ]
+        let elevationSource = AGSArcGISTiledElevationSource(url: elevationServiceURL)
         let surface = AGSSurface()
-        surface.elevationSources.append(contentsOf: elevationSources)
+        surface.elevationSources.append(elevationSource)
         scene.baseSurface = surface
         
-        /// The url of the scene service for buildings in Brest, France.
+        // The URL of the scene service for buildings in Brest, France.
         let brestBuildingsServiceURL = URL(string: "https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0")!
         // Create the building layer and add it to the scene.
         let buildingsLayer = AGSArcGISSceneLayer(url: brestBuildingsServiceURL)
+        // Offset the altitude to avoid clipping with the elevation source.
+        buildingsLayer.altitudeOffset = 2
```

Comment:
Last week most people agreed to add an offset to the buildings layer to mitigate the clipping between elevated surface and buildings. Once the design get finalized, I'll request another review from you all.